README.md

```diff
@@ -50,6 +51,10 @@
 
 - [jest-codemods](https://github.com/skovhus/jest-codemods) Makes it easy to migrate from other test runner and frameworks to Jest.
 
+### Library integrations
```

Comment:
I'm not sure about where else to put this, or if this title is any good...

Reply:
@searls ideas for "genre" here? 🙂

Reply:
Mocks maybe?

Reply:
Library plugins or extensions seems reasonable to me. One would think that quite a few of these probably do exist (and few having to do with mocking)